docs/Constraints.md

```diff
@@ -148,7 +149,7 @@ All the installed (queryable via `dotnet workload list`) as well as [extended](h
 - `type`: `workload`
 - `args` (string, array). Mandatory. List of names of supported workloads (running host need to have at least one of the requested workloads installed).
 
-  To see the list of instaled workloads run [`dotnet workload list`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-list). To see the list of available workloads run [`dotnet workload search`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-search)
+  To see the list of installed workloads run [`dotnet workload list`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-list). To see the list of available workloads run [`dotnet workload search`](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-search)
 
 **Supported in**:
    - .NET SDK CLI (`dotnet new`)
```

Comment:
This should be supported in 17.4 by the time we ship. I'm currently blocked due to the 7.0 template locator not being inserted yet.